client/lib/i18n-utils/switch-locale.js

```diff
@@ -18,14 +18,13 @@ const getPromises = {};
 /**
  * De-duplicates repeated GET fetches of the same URL while one is taking place.
  * Once it's finished, it'll allow for the same request to be done again.
- *
  * @param {string} url The URL to fetch
  *
  * @returns {Promise} The fetch promise.
  */
 function dedupedGet( url ) {
 	if ( ! ( url in getPromises ) ) {
-		getPromises[ url ] = globalThis.fetch( url ).finally( () => delete getPromises[ url ] );
+		getPromises[ url ] = fetch( url ).finally( () => delete getPromises[ url ] );
```

Comment:
should this fix stick around? ugh.

Reply:
Good call.  eslint will be unhappy otherwise.

Reply:
Well, lots of linter noise anyway due to the jsdoc issues.  Will merge and deal with that in the context of the actual fix.